modules/serverless-scaling-serving.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * /knative-serving/scalability-and-performance-serving.adoc
+
+
+:_content-type: CONCEPT
+[id="serverless-scaling-serving_{context}"]
+= Scaling and performance of OpenShift Serverless Serving
+
+You can configure the scaling of OpenShift Serverless Serving using the `KnativeServing` custom resource (CR). OpenShift Serverless Serving has to be scaled and configured based on the following parameters:
+
+* Number of Knative Services
+* Number of Revisions
+* Amount of concurrent requests in the system
+* Size of payloads of the requests
+* The startup-latency and response latency of the Knative Service added by the user’s web application
+* Number of changes of the KnativeService Custom Resources over time
+
+[id="serverless-scaling-serving-defaults_{context}"]
+== KnativeServing default configuration 
+
+Per default, OpenShift Serverless Serving is configured to run all components with high-availability and medium-sized CPU and memory requests/limits. This means that the `high-available` field in `KnativeServing` CR is automatically set to a value of `2` and all system components are scaled to two replicas. This configuration is suitable for medium workload scenarios and has been tested with:
```

Comment:
I kind of understand where this section is coming from, but it does feel a bit weird, as it's oddly specific in some aspects of it while also not being actually helpful to anybody...

Reply:
@maschmid  I think this should explain that the default is HA and sufficient medium sized installations, thus the below list showcasing what that means. If a customer has different load-pattern he probably needs to change something.

Do you have a specific rewording in mind that would fit better?